lib/srv/desktop/tdp/proto.go

```diff
@@ -1460,3 +1470,10 @@ func writeUint64(b *bytes.Buffer, v uint64) {
 	b.WriteByte(byte(v >> 8))
 	b.WriteByte(byte(v))
 }
+
+// tdpMaxNotificationMessageLength is somewhat arbitrary, as it is only sent *to*
+// the browser (Teleport never receives this message, so won't be decoding it)
+const tdpMaxNotificationMessageLength = 10240
+const tdpMaxPathLength = 10240
```

Comment:
I assume the comment regarding `tdpMaxNotificationMessageLength` also applies to `tdpMaxPathLength` ?

Reply:
Sort of... It's actually something I need to research further, previously we were setting it as `tdpMaxPathLength = tdpMaxNotificationMessageLength`, but I decided to change that here to make it clear that there's no relationship between the two values. 

Added a TODO comment that links to the relevant issue (https://github.com/gravitational/teleport/issues/14950)

294e91536467e416b5899597ce5ee08a457a13a9

Reply:
Great !